HTML＆CSS　追加修正

mike3104 opened the pull request:
いろいろまちがった使い方をしましたが、githubをどう使うのかがようやくわかってきた気がします。
HTML＆CSSですが、かくリーダーから的確なコメントをしていただけるので非常に助かってます。ありがとうございます。

以下修正いたしました。
 ①
フッターの右側は「お問い合わせ」と「利用規約」だけにしてもらえますか？
→まだ項目がきまっていないとおもったので、構成を考えるためにもいろいろ列挙してしました、、、
　採用情報は冗談がすぎましたね笑

 ②
・SNSの削除
・like-counterへの修正
・<nav>への<a></a>追加
→対応しました

③
ページの上下左右に少しだけだぶつきがあるのですが、原因はユーザーエージェントスタイルと思われます。body要素のにmargin: 0; を加えください。もしくはユーザーエージェントスタイルを無効化するのが一番いいと思うので、こちらのサイトを参照にやってみてください。ressがいいかも。
→ressをいれました。

④検索のレスポンシブ対応化

Commit:
HTML＆CSS　追加修正

いろいろまちがった使い方をしましたが、githubをどう使うのかがようやくわかってきた気がします。
かくリーダーから的確なコメントをしていただけるので、非常に助かります。ありがとうございます。

 ①
フッターの右側は「お問い合わせ」と「利用規約」だけにしてもらえますか？
→まだ項目がきまっていないとおもったので、構成を考えるためにもいろいろ列挙してしました、、、
　採用情報は冗談がすぎましたね笑

 ②
・SNSの削除
・like-counterへの修正
・<nav>への<a></a>追加
→対応しました

③
ページの上下左右に少しだけだぶつきがあるのですが、原因はユーザーエージェントスタイルと思われます。body要素のにmargin: 0; を加えください。もしくはユーザーエージェントスタイルを無効化するのが一番いいと思うので、こちらのサイトを参照にやってみてください。ressがいいかも。
→ressをいれました。

kakudaisuke commented:
ミケさん、ごめんなさい！僕が間違えてました。次から、修正の場合は新しいプルリクしなくて大丈夫です！！（ミケさんが合ってた）
修正おつかれさまです！てか、早(笑)
- [ ] 画面上下左右のだぶつきが直ってないですね。（borderが見えてしまってますよね。）
リセットCSSファイルを追加してくださったと思うのですが、多分読み込まれてないですね。
ただこの点も僕のアドバイスが不明確でした。headタグ内で、
`<link rel="stylesheet" href="styles.css">`
の上に
`<link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">`
を追加するだけで外部から読み込むこともできます。そっちの方がシンプルかと思うのでそうしませんか？
- [ ] リンクタグが必要な箇所にaタグの追加をした方があとが楽かもですね。nav内要素, footer内要素, サイトロゴなどです。

Commit:
再度変更しました。

変更しました。
・headerの配色範囲の修正
・<link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">の追加
→理解はできていませんが、とりいそぎ修正しました。
・aタグの追加
